docs/study-view/layout.md

```diff
@@ -0,0 +1,6 @@
+## How does the page layout the charts
+Study view page is fully responsive, it will try to fit more charts in the same height as many as possible base on the browser width.
```

Comment:
*based

Reply:
`it will try to fit as many charts as possible based on the browser's width and height`